membership-attribute-service/app/controllers/AccountController.scala

```diff
@@ -40,18 +42,18 @@ class AccountController extends LazyLogging {
 
     // TODO - refactor to use the Zuora-only based lookup, like in AttributeController.pickAttributes - https://trello.com/c/RlESb8jG
 
-    val updateForm = Form { single("stripeToken" -> nonEmptyText) }
+    val updateForm = Form { tuple("stripeToken" -> nonEmptyText, "publicKey" -> text) }
     val tp = request.touchpoint
     val maybeUserId = authenticationService.userId
     logger.info(s"Attempting to update card for $maybeUserId")
     (for {
-      user <- EitherT(Future.successful( maybeUserId \/> "no identity cookie for user"))
-      stripeCardToken <- EitherT(Future.successful(updateForm.bindFromRequest().value \/> "no card token submitted with request"))
+      user <- EitherT(Future.successful(maybeUserId \/> "no identity cookie for user"))
+      stripeDetails <- EitherT(Future.successful(updateForm.bindFromRequest().value \/> "no card token submitted with request"))
```

Comment:
We should tweak the error message here too, as it could be a missing token **or** public key